packages/core/src/Chip/FilterChip/FilterChip.js

```diff
@@ -6,7 +6,8 @@ import { ChipContent } from '../ChipContent'
 import { ChipLabel } from '../ChipLabel'
 import { ChipInput } from '../ChipInput'
 
-const getTitle = ({ disabled }) => (disabled ? 'Disabled' : 'Close')
+const getTitle = ({ disabled, actionTitle }) =>
+  disabled ? 'Disabled' : actionTitle
```

Comment:
Would it make sense to append the word `disabled` after the action title, so it would read something like `<action title> disabled`?

Reply:
Updated